Geometry/HGCalCommonData/plugins/dd4hep/DDHGCalEEAlgo.cc

```diff
@@ -369,7 +374,7 @@ struct HGCalEEAlgo {
         }
 #endif
         if (corner.first > 0) {
-          int type = waferType_->getType(xpos, ypos, zpos);
+          int type = waferType_->getType((xpos / dd4hep::mm), (ypos / dd4hep::mm), (zpos / dd4hep::mm));
```

Comment:
Could you please explain briefly why conversion to mm is being added here? I'm not understanding it.

Reply:
Inside the code it uses a fourth order  polynomial in z with coefficients rad100, rad200 and all these are calculated in mm. So it is better to pass on all parameters to the function in mm so that the function can work for both DDD and DD4Hep

Reply:
If I understand correctly, `HGCalWaferType` can work with either cm or mm, as long as they are used consistently. You are changing this code here to mm because that is what is used elsewhere in a similar context.